app/assets/stylesheets/_forms.scss

```diff
@@ -8,7 +8,7 @@ fieldset {
 input,
 label,
 select {
-  display: block;
+  display: inline;
```

Comment:
I had originally put selects as block but then I noticed that checkboxes were inline so I decided to go with this. I can revert back to `display: block;`